platform/provider.go

```diff
@@ -143,7 +143,7 @@ func NewProvider(logger boshlog.Logger, dirProvider boshdirs.Provider, statsColl
 			compressor,
 			copier,
 			dirProvider,
-			vitalsService,
+			linxuVitalsService,
```

Comment:
I think you may need to pull your bosh-agent code. Centos has been removed as a build target for the bosh-agent.